orchestration/hca_orchestration/tests/e2e/conftest.py

```diff
@@ -114,6 +114,13 @@ def copy_project_config():
                     "region": "US"
                 }
             }
+        },
+        "solids": {
+            "direct_copy_from_tdr": {
```

Comment:
Should this be the name of the function containing this parameter (ingest_data_files)?

Reply:
Yup, good catch.